src/index.js

```diff
@@ -0,0 +1,28 @@
+import apis from '../apis.json';
```

Comment:
hmm... I had issues with `require(...json)` in browserify, because it's not a `js` file..